src/react/ReportControls/components/SelectReportType/hooks/useReports/useReports.ts

```diff
@@ -0,0 +1,8 @@
+import { SetRouteData, useRouteData } from '../../../../../hooks/useRouteData';
+import { DueReport, StartDueReport, TableReport } from '../../utilities';
+
+export type Reports = [StartDueReport, DueReport, TableReport];
+
+export type SetReports = SetRouteData<Reports>;
+
+export const useReports = (): readonly [Reports, SetReports] => useRouteData<Reports>('reports');
```

Comment:
no change needed just a question

Are we gaining anything by defining the return type of `useReports` over just leaning on inference?

Reply:
minor optimization by leaning on explicitly defined types for exports. Loads an unnoticable amount of time faster on vscode